procrastinate/worker.py

```diff
@@ -25,6 +25,10 @@ def __init__(
         self.app.perform_import_paths()
         self.job_store = self.app.job_store
 
+        self.logger = logging.getLogger(
+            __name__ + f"[{name}]" if name is not None else ""
+        )
```

Comment:
I'm not sure it's a good practice to define a logger per object.
Maybe this can be done with a logging-context variable?

Reply:
I don't know either. I am not too shocked by this object-specific logger :)

Reply:
I think it would be better if each logger was the child of the « main » logger. Logger paternity is defined by dots in the name. Thus, procrastinate.worker.<name> may be better.

Though, that might be more easily achieved with logger.getChild():
https://docs.python.org/3/library/logging.html#logging.Logger.getChild